influxdb3_processing_engine/src/lib.rs

```diff
@@ -422,6 +422,20 @@ impl ProcessingEngineManager for ProcessingEngineManagerImpl {
         Ok(())
     }
 
+    async fn start_triggers(&self) -> Result<(), ProcessingEngineError> {
+        let triggers = self.catalog.active_triggers();
+        for (db_name, trigger_name) in triggers {
+            self.run_trigger(
+                Arc::clone(&self._write_buffer),
+                Arc::clone(&self._query_executor),
```

Comment:
since these are now being used, we can remove the leading `_` that was there to get past the lint?